js/modules/k6/html/html.go

```diff
@@ -44,17 +47,201 @@ type Selection struct {
 	sel *goquery.Selection
 }
 
-func (s Selection) Add(arg goja.Value) Selection {
-	switch val := arg.Export().(type) {
+func (s Selection) emptySelection() Selection {
+	// Ask for out of bounds item for an empty selection.
+	return s.Eq(s.Size())
+}
+
+func (s Selection) buildMatcher(v goja.Value, gojaFn goja.Callable) func(int, *goquery.Selection) bool {
+	return func(idx int, sel *goquery.Selection) bool {
+		fnRes, fnErr := gojaFn(v, s.rt.ToValue(idx), s.rt.ToValue(sel))
+
+		if fnErr != nil {
+			panic(fnErr)
+		}
+
+		return fnRes.ToBoolean()
+	}
+}
+
+func (s Selection) varargFnCall(arg interface{},
```

Comment:
Changed from goja.Value to interface{} her to fix a problem I ran into last night (not sure how I missed it :\) at line 87/80 of http.go had been using a string selector to Find(). I could have changed Find to a string only - or - replace the goja.Value with an interface{} and type switch as before, I went with the interface{} and had to use exportIfGojaVal. 

Feel uncertain about this (partly because I didn't spot the problem earlier!)...